### PR TITLE
rdp controller described, .gitbook/assets folder renamed to -assets

### PR DESCRIPTION
**Pull request by Vegebutcher**
In RDP Controller we should give the link to the .reg file. We should decide where it should be: Manual assets? Docs assets? Elsewhere?

### CODE REVIEW

File: appendices/rdp-controller.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# RDP Controller
+
+RDP stands for Remote Desktop Protocol and is the foundation for remote desktop connections in Windows. With RDP you can connect to a remote computer running Windows and use it as if it was your local machine.
+
+Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.
+
+In order to start using the RDP Controller, you have to follow these steps:
+
+1. Download and run [this file]() on your local Windows system, answering *Yes* to the security warning message.
```
Comment (WiktoriaPrusik) [inline appendices/rdp-controller.md]: `[this file]()` 
there's no path to this file

File: appendices/rdp-controller.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+4. Launch G1ANT.Robot, go to `Tools/Installers` menu and select `RDP Controller`.
+
+5. Repeat step 3. and enter these credentials in the RDP Configuration dialog box.
+
+6. Click Yes in the security warning message box. Wait until the command line installer finishes its work and when it does, press any key to close the command line window.
+
+   To check if the RDP Controller was successfully installed, open Windows Task Manager (press **Ctrl+Shift+Esc**) go to Services tab and look for *Disconnected Rdp Session Replacer* service. If it’s there and running, the RDP Controller is ready for use.
+
+   ![](G1ANT.Manual/-assets/rdp-task.jpg)
```
Comment (WiktoriaPrusik) [inline appendices/rdp-controller.md]: incorrect path

Comment (WiktoriaPrusik) [reply appendices/rdp-controller.md]: should be:  ``
it's always good to check if something is visible on github by clicking view file

File: appendices/rdp-controller.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.
+
+In order to start using the RDP Controller, you have to follow these steps:
+
+1. Download and run [this file]() on your local Windows system, answering *Yes* to the security warning message.
+
+2. Launch Remote Desktop Connection tool in Windows: type `remote` in the Windows Search box (*Windows 7:* click Start Menu, then use the search box; *Windows 8 and above:* click the Search icon on the taskbar) and click the name of the app in the search results. You can also select the tool in the Accessories (Windows 7) or Windows Accessories (Windows 8 and above) folder of the Start menu (click All Programs first in case of Windows 7).
+
+   ![](G1ANT.Manual/-assets/rdc-start.jpg)
```
Comment (WiktoriaPrusik) [inline appendices/rdp-controller.md]: incorrect path, should be: ``

**Review by WiktoriaPrusik — CHANGES_REQUESTED** [review]
I get that `.gitbook` folder is unnecessary here but why did you rename `assets` to `-assets`?
It should be also synched by my tool to G1ANT.Docs as these files will be needed there

File: appendices/rdp-controller.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# RDP Controller
+
+RDP stands for Remote Desktop Protocol and is the foundation for remote desktop connections in Windows. With RDP you can connect to a remote computer running Windows and use it as if it was your local machine.
+
+Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.
```
Comment (pwitusik) [inline appendices/rdp-controller.md]: Try something like that:
"Our RDP Controller is a tool allows running automations with G1ANT.Robot on a remote PC without disturbance even when you close you RDP client and disconnect RDP session..."

File: appendices/rdp-controller.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Download and run [this file]() on your local Windows system, answering *Yes* to the security warning message.
+
+2. Launch Remote Desktop Connection tool in Windows: type `remote` in the Windows Search box (*Windows 7:* click Start Menu, then use the search box; *Windows 8 and above:* click the Search icon on the taskbar) and click the name of the app in the search results. You can also select the tool in the Accessories (Windows 7) or Windows Accessories (Windows 8 and above) folder of the Start menu (click All Programs first in case of Windows 7).
+
+   ![](G1ANT.Manual/-assets/rdc-start.jpg)
+
+3. Enter username and password of the user on the remote machine.
+
+4. Launch G1ANT.Robot, go to `Tools/Installers` menu and select `RDP Controller`.
```
Comment (pwitusik) [inline appendices/rdp-controller.md]: Add information that to install RDP Controller G1ANT.Robot needs admin privileges.

File: appendices/rdp-controller.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+   To check if the RDP Controller was successfully installed, open Windows Task Manager (press **Ctrl+Shift+Esc**) go to Services tab and look for *Disconnected Rdp Session Replacer* service. If it’s there and running, the RDP Controller is ready for use.
+
+   ![](G1ANT.Manual/-assets/rdp-task.jpg)
+
+7. Run your script.
+
+Now you can minimize the remote desktop window or close it altogether.
+
+**Note:** the two Registry keys, which are added by our file installed in step 1, allow minimizing the remote desktop window without affecting the robot and its script execution.
```
Comment (pwitusik) [inline appendices/rdp-controller.md]: Please add note that without those registry entries automation will not work correctly when RDP client will be minimised

**Review by pwitusik — CHANGES_REQUESTED** [review]
check out my comments

**Review by folbo — CHANGES_REQUESTED** [review]
Fix paths to images in **all** referencing md files.
for example, you changed `.gitbook/assets/main.jpg → -assets/main.jpg` but g1ant.robot-window/overview.md (which has reference to old path) is untouched in this PR.

File: appendices/rdp-controller.md (hunk 1 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]: don't you say :smile:
look at my first comment

File: appendices/rdp-controller.md (hunk 2 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]:
```suggestion
   ![](../-assets/rdp-task.jpg)
```

File: appendices/rdp-controller.md (hunk 3 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]:
```suggestion
   ![](../-assets/rdc-start.jpg)
```

File: appendices/rdp-controller.md (hunk 4 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]: The sense is almost the same, only my description is more detailed and probably more comprehensive for people who are not that advanced in IT.

File: appendices/rdp-controller.md (hunk 5 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]:
```suggestion
4. Launch G1ANT.Robot, go to `Tools/Installers` menu and select `RDP Controller`. You will need administrator rights to install the RDP Controller.
```

File: appendices/rdp-controller.md (hunk 6 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]:
```suggestion
**Note:** the two Registry keys, which are added by our file installed in step 1, allow minimizing the remote desktop window without affecting the robot and its script execution. Without these registry entries the automation will not work correctly when the RDP client is minimized.
```

**Comment by Vegebutcher** [comment]
@folbo links changed in md files from Manual repo, as requested

File: appendices/rdp-controller.md (hunk 4 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]:
```suggestion
Our RDP Controller is a tool for running G1ANT.Robot automations on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection client. The script will continue the execution without disturbance.
```

**Comment by lfronczyk** [comment]
> In RDP Controller we should give the link to the .reg file. We should decide where it should be: Manual assets? Docs assets? Elsewhere?

The file's placed in the appendices, so put it there, if it's not possible then put it wherever you want (ofc use common sense ;)). For me it can be a docs asset.

**Review by lfronczyk — APPROVED** [review]

File: appendices/rdp-controller.md (hunk 1 above)
Comment (Vegebutcher) [reply appendices/rdp-controller.md]:
```suggestion
1. Download and run [this file](../-assets/RDP local machine.reg) on your local Windows system, answering *Yes* to the security warning message.
```

**Pull request merged**